example-analysis-scripts/pyutils-examples/example_cuts.py

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env python
```

Comment:
I've been putting rooutil examples under ```rooutil/```. I think I'd like to get rid of the top-level ```example-analysis-scripts``` directory but perhaps it's good to have something obvious at the top-level for people to see...

Reply:
good point, I think examples makes most sense in the example-analysis-script and keep the utils as their own thing but upto you. Let me know and I will move

Reply:
Yeh, I think that's right. Do we also want to move the invidual utils to the top-level? So we would have something like:
```
EventNtuple/
| examples/
| inc/
| src/
| pyutil/
| rooutil/
| validation/
```

Reply:
I think that could work, they are separate entities

Reply:
Sounds good. Let's do that in the next PRs. I'll merge this now